spec/factories/ips.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :ip do
+    address "1.12.123.45.78"
```

Comment:
A very strange IP, think once we add the validation, these specs will start failing.